golem-worker-executor-base/src/durable_host/durability.rs

```diff
@@ -268,13 +268,13 @@ impl<Ctx: WorkerCtx> DurableWorkerCtx<Ctx> {
                     serializable_result
                 )
             });
-            self.state.set_oplog_entry(oplog_entry).await;
+            self.state.oplog.add(oplog_entry).await;
```

Comment:
Is this a logical change required for the persistent queue, or just refactor?

Reply:
Refactoring how the oplog is accessed was necessary, because now we need to write to it outside of the worker's context.